likeminds-chat-reactnative-integration/ChatSX/components/MessageList/index.tsx

```diff
@@ -463,22 +485,22 @@ const MessageListComponent = ({
             inverted
           />
           {isScrollingUp && (
-            <TouchableOpacity
-              style={[
-                styles.arrowButton,
-                {
-                  bottom: keyboardVisible
-                    ? Layout.normalize(55)
-                    : Layout.normalize(20),
-                },
-              ]}
-              onPress={scrollToBottomProp ? scrollToBottomProp : scrollToBottom}
-            >
-              <Image
-                source={require("../../assets/images/scrollDown.png")}
-                style={styles.arrowButtonImage}
-              />
-            </TouchableOpacity>
+            <SafeAreaView style={{zIndex: 10}}>
+              <TouchableOpacity
+                style={[
+                  styles.arrowButton,
+                  {
+                    bottom: Layout.normalize(-50)
```

Comment:
why styling becomes -55 now? Is this tested properly?

Reply:
had to try with different values, have tested this on SE, Iphone 16, and 3 other android devices